packages/form-js-editor/src/features/modeling/FormLayoutUpdater.js

```diff
@@ -0,0 +1,53 @@
+import { updateRow } from './cmd/Util';
+
+import CommandInterceptor from 'diagram-js/lib/command/CommandInterceptor';
+
+import { clone } from '@bpmn-io/form-js-viewer';
+
+
+export default class FormLayoutUpdater extends CommandInterceptor {
+
+  constructor(eventBus, formLayouter, modeling, formEditor) {
+    super(eventBus);
+
+    this._eventBus = eventBus;
+    this._formLayouter = formLayouter;
+    this._modeling = modeling;
+    this._formEditor = formEditor;
+
+    // @ts-ignore
```

Comment:
Why is this necessary?

Reply:
Let me double check, it could be that the missing types of `diagram-js` were striking here.

Reply:
Yes indeed, it's due to lacking diagram-js types. We have similar occasions in the code base (modeling behaviors, command interceptors).